### ISSUE TITLE
After deleting all segmentstore pods all reader moves to inactive state

### ISSUE THREAD

**Issue by Ranjan-Padhi** [pravega/pravega#4216]
After deploying pravega 0.6.0-2369.d4a22be release, started Benchmark client IO with below request 

` ./pravega-benchmark -producers 5 -controller tcp://10.100.200.30:9090 -events 100 -time 18000 -segments 50 -size 100 -stream Stream26thA -consumers 5`

while IO running write rate displaying as 10 k/s and read rate displaying 10 k/s in grafana dashboard for the stream
Now delete all segmentstore pods, after all pods came up IO recovered immediate(within a minute) but read rate reduce to 0 KiB/s and not comeback even after 20 mins, and no reading record in client

Its look like after deleting all segmentstore pods all readers are move to inactive state only writing is happening

```
Note :- I verified same scenario 3 times
In first 2 times, read rate reduce slowly toward 0 KiB/s after deleting all segmentstore pods 2-3 times (in first delete read rate reduce to around 8 KiB/s, then second delete read rate reduce to 4 KiB/s and third delete read rate reduce to 0 KiB/s)

But this 3 time where read rate directly move to 10 KiB/s to 0 KiB/s in first deleting all segmentstore pods
```
Environment details: PKS / K8 with medium cluster:
```
3 master nodes @ large.cpu (4 CPU, 4 GB Ram, 16 GB Disk)
5 worker nodes @ xlarge.cpu(8 cpu, 8 GB Ram, 32 GB Disk)
Tier-1 storage is from VSAN datastore
Tier-2 storage curved on NFS Client provisioner using Isilon as backend
```
Deployment Details:-
```
bookkeeper:0.6.0-2369.d4a22be
pravega:0.6.0-2369.d4a22be
pravega-operator:0.4.2-rc1
pravega/zookeeper-operator:0.2.2
```

**Comment by Ranjan-Padhi** [comment]
![Grafana_Dashboard_Read_Write](https://user-images.githubusercontent.com/48913765/65714146-2d4aec80-e0b8-11e9-958b-a531671a143d.jpg)
![Grafana_Dashboard](https://user-images.githubusercontent.com/48913765/65714149-2d4aec80-e0b8-11e9-811f-29b659456fc4.jpg)

**Comment by Ranjan-Padhi** [comment]
```
default       pravega-bookie-0                                                1/1     Running     0          106m
default       pravega-bookie-1                                                1/1     Running     0          106m
default       pravega-bookie-2                                                1/1     Running     0          106m
default       pravega-bookie-3                                                1/1     Running     0          106m
default       pravega-operator-7d786695bd-nhbcp                               1/1     Running     0          107m
default       pravega-pravega-controller-56cf4c94-6td7f                       1/1     Running     0          106m
default       pravega-pravega-controller-56cf4c94-p6794                       1/1     Running     0          106m
default       pravega-pravega-segmentstore-0                                  1/1     Running     0          64m
default       pravega-pravega-segmentstore-1                                  1/1     Running     0          64m
default       pravega-pravega-segmentstore-2                                  1/1     Running     0          64m
default       pravega-zk-0                                                    1/1     Running     0          111m
default       pravega-zk-1                                                    1/1     Running     0          110m
default       pravega-zk-2                                                    1/1     Running     0          110m
default       pravega-zk-3                                                    1/1     Running     0          109m
default       pravega-zk-4                                                    1/1     Running     0          109m
default       shaka-zulu-nfs-client-provisioner-867468f976-cklf8              1/1     Running     0          107m
default       zookeeper-operator-5946b99494-rzzjk                             1/1     Running     0          111m
```

**Comment by shrids** [comment]
From the logs observing that the readers failed due to authentication errors. (BenchMark Client log)
Log file : 
[Benchmark_Client_log.zip](https://github.com/pravega/pravega/files/3807633/Benchmark_Client_log.zip)

e.g:
 
```
Line 68313 [grpc-default-executor-17] WARN io.pravega.client.segment.impl.AsyncSegmentInputStreamImpl - Closing connection to segment Scope/Stream26thA/6.#epoch.0 with exception: io.pravega.shared.protocol.netty.ConnectionFailedException: java.util.concurrent.CompletionException: io.grpc.StatusRuntimeException: UNAUTHENTICATED: Authentication failed
io.grpc.StatusRuntimeException: UNAUTHENTICATED: Authentication failed
```

**Comment by ravisharda** [comment]
This issue was fixed in PR #4336. Accordingly, closing this issue.
